tests/vmi_tpm_test.go

```diff
@@ -83,3 +89,86 @@ var _ = Describe("[sig-compute]vTPM", func() {
 		})
 	})
 })
+
+var _ = Describe("[sig-storage]vTPM", func() {
+	var virtClient kubecli.KubevirtClient
+	var err error
+
+	BeforeEach(func() {
+		virtClient, err = kubecli.GetKubevirtClient()
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	Context("with persistent TPM VM option enabled", func() {
+		It("should persist TPM secrets across restarts", func() {
+			By("Setting nfs-csi as the backend storage class")
+			kv := util.GetCurrentKv(virtClient)
+			kv.Spec.Configuration.BackendStorageClass = "nfs-csi"
```

Comment:
Don't we need to skip the test `nfs-csi` storage class isn't installed?

Reply:
Indeed, will do.

Reply:
I switched to using the proper libstorage function and added a skip.